research/evm_explorer_comparison.md

```diff
@@ -0,0 +1,152 @@
+
+- **Additional Services:**
+	Beyond basic blockchain exploration, Etherscan offers a suite of additional features that enhance its utility. Such features are: [Dex tracker](https://etherscan.io/dex), [Gas tracker](https://etherscan.io/gastracker), [Token tracker](https://etherscan.io/tokens), [Smart contract verification](https://etherscan.io/verifyContract), [Developer tools](https://etherscan.io/directory/Others/Tools).
+
+**Cons**
+- **Proprietary:**
+
+	Etherscan is not open-source, hence its code is not publicly available for anyone to inspect or modify. For developers, this lack of transparency can be a drawback because it limits the ability to customize the explorer to suit specific needs or integrate it more deeply with their own projects.
+
+- **Ethereum-Centric:**
+
+	Etherscan is designed specifically for the Ethereum blockchain. While it excels in providing detailed information and tools for Ethereum, its support for other blockchain networks is very limited. Developers working with multiple blockchains might find this focus restrictive if they need a versatile tool that can handle different chains equally well.
+
+- **Limited Customization:**
+
+	Since Etherscan is a closed platform, users cannot modify or extend its functionality. Developers looking for specific features or integrations that aren't already provided by Etherscan will find themselves constrained by what the platform offers out of the box. This lack of flexibility can be a significant limitation for projects that require tailored solutions or advanced custom features.
+
+## Blockscout: Overview, Features, Pros, and Cons
+**Overview**
+
+[Blockscout](https://www.blockscout.com/) is an open-source block explorer made for Ethereum and other blockchains that use the Ethereum Virtual Machine (EVM). It’s built to be transparent and customizable, giving developers and users a flexible tool for looking at blockchain data.
+![Screenshot of Blockscout Homepage](img/blockscout_img.png)*Blockscout homepage*
+
+**Features**
+- **Open-Source:**
+
+	Unlike many other explorers that are closed systems, Blockscout's [code](https://github.com/blockscout/blockscout) is publicly accessible. This means anyone can examine, modify, or improve its features, there by making available a collaborative environment for development.
+
+- **Multi-Chain Support:**
+
+	Its versatility is another standout feature. Blockscout isn't confined to just one blockchain; it can explore multiple networks built on Ethereum's technology, offering a comprehensive view of the broader ecosystem.
+
+- **Customization:**
+
+	Users can tweak and adjust the explorer to meet their specific needs, implementing the interface and data display to their preferences.
+
+- **Detailed Information:**
```

Comment:
Does Blockscout stand out when compared to Etherscan ? I don't think so... I suggest removing